### ISSUE TITLE
Google Music Downloader has stopped working

### ISSUE THREAD

**Issue by cnpellegrino** [coman3/Google.Music.Downloader#2]
Everytime I log in to the application it crashes and I get Google Music Downloader has stopped working.

2 step authentication is off. 

I've tried Setup and Standalone verisons.

I'm on a windows 10 system

**Comment by lzmichael** [comment]
Same here. Did you got fixed it.

If yes, how you did it ?

**Comment by cnpellegrino** [comment]
I never figured it out completely but I did figure out it's something with my login security or something. My friend was able to use it on his computer perfectly. I logged in using his computer and it crashes. Had him login with his account on my computer and it works perfectly. i try logging in and it crashes. If you figure it out please let me know. Thanks!!

**Comment by lzmichael** [comment]
Thanks i play now a little around with my google-account settings...i let you know if i got it working...

For Windows there is no other tool available like this ?

**Comment by cnpellegrino** [comment]
There's another application that I found but it costs $$$ http://songspout.com/

I currently use an android app called musicmate that works really well but its a pain to transfer music between my android device and my PC.

**Comment by lzmichael** [comment]
I tried now with 3 different gmail accounts....and always it always crashes....2factor auth is disabled everywhere....i tried to start with "admin rights"....but no luck :(

**Comment by lzmichael** [comment]
I have found the "bug" now. you need to create a PLAYLIST in your google-music account then the tool works fine :)

if you dont have created a playlist the app crashes after login.

**Comment by lzmichael** [comment]
Since few days it saves the MP3's only in 160kbps, and not 320kbps anymore :(

Anyone have the same problem ? I tried with different accounts...and only 160kbps anymore...

**Comment by onelouder6** [comment]
I do have the same problem with 160kbps.

BTW coman3, I love you for making this app <3. Was _very_ hard to find and I was looking for it a long time  :)

**Comment by lzmichael** [comment]
Thanks, if you maybe have found out, how to save in 320kbps again, let us know :)

**Comment by foxt** [comment]
![Works fine for me](https://cdn.discordapp.com/emojis/323300460708757514.png)
![image](https://user-images.githubusercontent.com/12257112/30558284-d58432cc-9ca8-11e7-910d-90ac575d3f10.png)

**Comment by coman3** [comment]
Just for anyone, this error is occurred when the logged in user does not have any custom playlists.... Might fix up the error in the next couple of days as i should have time 😄

**Comment by Alia5** [comment]
I've tried creating a playlist, app still crashes on me :(

**Comment by coman3** [comment]
@Alia5 do you have 2 factor auth enabled? maybe try and set a "app specific password" [Check Here](https://support.google.com/accounts/answer/185833?hl=en)?

**Comment by Alia5** [comment]
Nope, no 2 factor here... Can't use and app specific PW 'cause of that.

Will prob. have to check myself once I get the time

**Comment by Alia5** [comment]
Found the issue...
I Never thumbs-upped a song ;)

**Comment by coman3** [comment]
ah that makes sense! Seeing this was supposed to be just for example use i never bothered with null checks 😜
